Add tips section for tooltip components (#6589)
yamada-ui/yamada-ui: www/contents/components/(components)/tooltip.mdx

<Tooltip />
```

:::tip

> If you need to display supplementary information with an icon button trigger, use [Tip](/docs/components/tip).

Reviewer: Align this with the existing `Button` tip and describe the condition by the `Tooltip` child instead of the icon button trigger.

```suggestion
If the only child of `Tooltip` is an icon, use [Tip](/docs/components/tip).
```

Author: Adressed in [13d4340](https://github.com/yamada-ui/yamada-ui/pull/6589/commits/13d4340dbb378ace017f55c67c8085fc042a22b1).

> If you want to display supplementary information using an icon button as a trigger, use [Tip](/docs/components/tip).

Reviewer 2: The guidance is correct, but this English reads a little awkward. Since `Tip` provides the icon button trigger itself, I think this can be phrased more directly.

```suggestion
If you need to display supplementary information with an icon button trigger, use [Tip](/docs/components/tip).
```

Author: Thanks for the suggestion! I think this wording is clearer.
